### ISSUE TITLE
Add volume adjustment for item refill sound [feature request]

### ISSUE THREAD

**Issue by RashidFTW** [Siphalor/mouse-wheelie#259]
**Is your feature request related to a problem? Please describe.**
Yes, the current "plop" is just too loud. It becomes even more deafening when dropping multiple stacks which triggers the auto-refill in quick successions .

**Describe the solution you'd like**
Add a volume adjustment slider (or a textbox where you can enter float percentage) for item refill sound.

**Describe alternatives you've considered**
Not that I tried..., but adjusting the vanilla sound or the overall pc sound is a no go. Currently all other block placing breaking sounds are balanced (most are already lowered).

**Additional context**
-

**Comment by FictionalXCI** [comment]
Yes. Please. The new sound is absolutely awful. I just updated to 1.20.4 and this sound is so startling and terrible. Please remove it.
